src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -121,12 +121,12 @@ public async Task<ResourceDownloadResults> EnsureResourceAsync(IHostEnvironment
             for (int i = 0; i < retryTimes; ++i)
             {
                 var thisDownloadResult = await DownloadFromUrlAsync(env, ch, url, fileName, timeout, filePath);
-
                 if (string.IsNullOrEmpty(thisDownloadResult))
                     return thisDownloadResult;
                 else
+                {
```

Comment:
Please follow convention. (It is non-standard...but) In the ML.NET codebase, when an if/else statement has just a single line, it is not enclosed in braces. Just like the previous if statement.